### PR TITLE
Put leader and disk usage on same line

### PR DESCRIPTION
**Pull request by ineiti**
image

### CODE REVIEW

File: dashboards-to-provision/services.json [hunk 1]
```diff
@@ -1639,7 +1639,7 @@
     "list": []
   },
   "time": {
-    "from": "now-3h",
+    "from": "now-7d",
```
Comment (tharvik) [inline dashboards-to-provision/services.json:1642]: You're sure?

Comment (ineiti) [reply dashboards-to-provision/services.json:1642]: Probably a personal preference, but I like to have a 1-week overview... We can take the inverse geometric mean of mine, yours, and @cgrigis preference ;)

@cgrigis what is your preference for the time-frame of stats.c4dt.org?

Comment (cgrigis) [reply dashboards-to-provision/services.json:1642]: I would personally put the last 24h :smile:
Then again it is easily adjustable with the dropdown, and 3h possibly loads faster?

Comment (ineiti) [reply dashboards-to-provision/services.json:1642]: So:

```
(3 * 24 * 168) ^ 1/3 = 23
```

I'd say we round up to 24 hours... So the geometric mean of 3 and 168 is about 24h.

**Review by tharvik — APPROVED** [review]

**Pull request merged**